SwrveSDK/SDK/Talk/SwrveBaseCampaign.m

```diff
@@ -251,6 +257,30 @@ -(BOOL)checkCampaignRulesForEvent:(NSString*)event
     return TRUE;
 }
 
+-(BOOL)checkCampaignTriggersForEvent:(NSString*)event
```

Comment:
Rename to:
 (BOOL) canTriggerWithEvent:(NSString *) event andPayload:(NSDictionary *)payload;
This will make it consistent with android.